hvac/api/secrets_engines/pki.py

```diff
@@ -798,6 +798,30 @@ def list_issuers(self, mount_point=DEFAULT_MOUNT_POINT):
             url=api_path,
         )
 
+    def set_issuers(self, params, mount_point=DEFAULT_MOUNT_POINT):
```

Comment:
This method should be probably be named `set_issuers_configuration` since it corresponds to that API: https://developer.hashicorp.com/vault/api-docs/secret/pki#set-issuers-configuration

We also don't want to take a generic `params` parameter. Instead we should accept a parameter called `default` (or maybe `default_issuer` to be more specific), and a `default_follows_latest_issuer` parameter so that these can be validated and documented properly.

The tests should then reflect these changes and test for success and failure conditions.

Reply:
Using `set_issuers_configuration` vs. `set_issuers` is somewhat inconsistent with `list_issuers` (titled [_Read issuers configuration_](https://developer.hashicorp.com/vault/api-docs/secret/pki#read-issuers-configuration) in the API docs, by the way).

Reply:
`default_follows_latest_issuer` is optional in the HTTP API. Taking it as an argument with an implicit default would probably unintentionally change the configured value.